lib/oath/views/account_view.dart

```diff
@@ -99,6 +99,24 @@ class _AccountViewState extends ConsumerState<AccountView> {
     }).toList();
   }
 
+  String? a11yCredentialLabel(String? issuer, String? name, OathCode? code) {
+    String? label = '';
+    String? tmpIssuer = issuer;
+    String? tmpName = name;
+    String? tmpCode = code?.value;
+    if (tmpIssuer != null) {
+      label += tmpIssuer;
+    }
+    if (tmpName != null) {
+      label += tmpName;
+    }
+    if (tmpCode != null) {
+      label += tmpCode;
+    }
+
+    return label;
+  }
+
```

Comment:
A lot of the `String?` types here don't need to be nullable. I'd change the return type, as well as the type of `name` and `label` to `String`. I also don't think any of the `tmp` variables are needed here.

Reply:
Agreed!